File: app/src/main/java/se/asapehrsson/podtest/model/Channel.java
package se.asapehrsson.podtest.model;

public class Channel {

    private String image;
    private String imagetemplate;
    private String color;
    private String siteurl;
    private Liveaudio liveaudio;
    private String scheduleurl;
    private String channeltype;
    private String xmltvid;
    private Integer id;
    private String name;

    public String getImage() {
        return image;
    }

    public String getImagetemplate() {
        return imagetemplate;
    }

    public String getColor() {
        return color;
    }

    public String getSiteurl() {
        return siteurl;
    }

    public Liveaudio getLiveaudio() {
        return liveaudio;
    }

    public String getScheduleurl() {
        return scheduleurl;
    }

    public String getChanneltype() {
        return channeltype;
    }

    public String getXmltvid() {
        return xmltvid;
    }

    public Integer getId() {
        return id;
    }

    public String getName() {
        return name;
    }
}

File: app/src/main/java/se/asapehrsson/podtest/data/Channel.kt
package se.asapehrsson.podtest.data

import com.google.gson.annotations.SerializedName

data class Channel(
        @SerializedName("image") val image: String? = null,
        @SerializedName("imagetemplate") val imagetemplate: String? = null,
        @SerializedName("color") val color: String? = null,
        @SerializedName("siteurl") val siteurl: String? = null,
        @SerializedName("liveaudio") val liveaudio: Liveaudio? = null,
        @SerializedName("scheduleurl") val scheduleurl: String? = null,
        @SerializedName("channeltype") val channeltype: String? = null,
        @SerializedName("xmltvid") val xmltvid: String? = null,
        @SerializedName("id") val id: Int? = null,
        @SerializedName("name") val name: String? = null
)